### PR TITLE
Remove specified target

### PR DESCRIPTION
**Pull request by velocity303**
Since the host types have sane default targets for *nix and Windows operating systems, this module can be used cross platform if this target is not hard coded. If this functionality is desired however, this can still be established with resource defaults set elsewhere (such as site.pp)

### CODE REVIEW

File: manifests/init.pp [hunk 1]
```diff
@@ -113,6 +108,12 @@
     host_aliases => $my_localhost_aliases,
     ip           => $localhost_ip,
   }
+  
+  host { "${::fqdn}" :
```
Comment (ghoneycutt) [inline manifests/init.pp]: This seems totally unrelated to the target, could you open a new PR or issue explaining this?

File: manifests/init.pp [hunk 2]
```diff
@@ -121,9 +122,9 @@
   }
 
   if $use_fqdn_real == true {
-    @@host { $::fqdn:
```
Comment (ghoneycutt) [inline manifests/init.pp]: same as above

File: manifests/init.pp [hunk 3]
```diff
@@ -15,7 +15,6 @@
   $localhost6_aliases    = ['localhost6',
                             'localhost6.localdomain6'],
   $purge_hosts           = false,
-  $target                = '/etc/hosts',
```
Comment (ghoneycutt) [inline manifests/init.pp]: This would need to stay so that people may specify a target, though it should default to `undef` so that the provider can pick the correct one.

File: manifests/init.pp [hunk 4]
```diff
@@ -100,10 +99,6 @@
     $fqdn_ip              = $::ipaddress
   }
 
-  Host {
```
Comment (ghoneycutt) [inline manifests/init.pp]: This should stay, but be wrapped in a conditional if `$target` is not `undef`.

**Comment by ghoneycutt** [comment]
Hi @velocity303 Thanks for the input, though I have a few questions.

Noticed you are working off of the master branch. You should create a new topic branch for each idea and send PR's from those.

**Comment by velocity303** [comment]
Yep, looks like that's why there are a few extra commits here than what I intended. I'll create a new topic branch and send a cleaned up pull request. Sorry for the confusion.
